financial-connections/src/main/java/com/stripe/android/financialconnections/di/FinancialConnectionsSheetNativeModule.kt

```diff
@@ -70,6 +71,14 @@ internal interface FinancialConnectionsSheetNativeModule {
             stripeNetworkClient = stripeNetworkClient
         )
 
+        @Provides
+        @Singleton
+        fun providesCustomTabsManager(
```

Comment:
Can this be done with a `@Binds` instead, as we already have the logger in the dependency graph?

Reply:
https://github.com/stripe/stripe-android/pull/7081#pullrequestreview-1547981011:~:text=Uses%20binds.-,025d0b9,-carlosmuvi%2Dstripe%20added